src/actions/bots.test.js

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import assert from 'assert';
+import nock from 'nock';
+
+import * as BotActions from 'actions/bots';
+import {Client4} from 'client';
+
+import TestHelper from 'test/test_helper';
+import configureStore from 'test/test_store';
+
+describe('Actions.Bots', () => {
+    let store;
+    beforeAll(async () => {
+        await TestHelper.initBasic(Client4);
+    });
+
+    beforeEach(async () => {
+        store = await configureStore();
+    });
+
+    afterAll(async () => {
+        await TestHelper.tearDown();
+    });
+
+    it('loadBots', async () => {
```

Comment:
nit: should this test load multiple (2?) fake bots?

Reply:
fixed.